From: Ben Hutchings <ben@decadent.org.uk>
Date: Fri, 13 Apr 2018 20:10:28 +0100
Subject: wireless: Add Debian wireless-regdb certificates
Forwarded: not-needed

This hex dump is generated using:

{
    for cert in debian/certs/wireless-regdb-*.pem; do
        openssl x509 -in $cert -outform der;
    done
} | hexdump -v -e '1/1 "0x%.2x," "\n"' > net/wireless/certs/debian.hex

---
Index: linux/net/wireless/certs/debian.hex
===================================================================
--- /dev/null
+++ linux/net/wireless/certs/debian.hex
@@ -0,0 +1,944 @@
+0x30,
+0x82,
+0x03,
+0xac,
+0x30,
+0x82,
+0x02,
+0x94,
+0xa0,
+0x03,
+0x02,
+0x01,
+0x02,
+0x02,
+0x09,
+0x00,
+0xd9,
+0xb1,
+0xe4,
+0x6f,
+0x28,
+0xf3,
+0xd4,
+0x97,
+0x30,
+0x0d,
+0x06,
+0x09,
+0x2a,
+0x86,
+0x48,
+0x86,
+0xf7,
+0x0d,
+0x01,
+0x01,
+0x0b,
+0x05,
+0x00,
+0x30,
+0x6b,
+0x31,
+0x0b,
+0x30,
+0x09,
+0x06,
+0x03,
+0x55,
+0x04,
+0x06,
+0x13,
+0x02,
+0x47,
+0x42,
+0x31,
+0x13,
+0x30,
+0x11,
+0x06,
+0x03,
+0x55,
+0x04,
+0x07,
+0x0c,
+0x0a,
+0x4d,
+0x61,
+0x6e,
+0x63,
+0x68,
+0x65,
+0x73,
+0x74,
+0x65,
+0x72,
+0x31,
+0x0f,
+0x30,
+0x0d,
+0x06,
+0x03,
+0x55,
+0x04,
+0x0a,
+0x0c,
+0x06,
+0x44,
+0x65,
+0x62,
+0x69,
+0x61,
+0x6e,
+0x31,
+0x16,
+0x30,
+0x14,
+0x06,
+0x03,
+0x55,
+0x04,
+0x03,
+0x0c,
+0x0d,
+0x42,
+0x65,
+0x6e,
+0x20,
+0x48,
+0x75,
+0x74,
+0x63,
+0x68,
+0x69,
+0x6e,
+0x67,
+0x73,
+0x31,
+0x1e,
+0x30,
+0x1c,
+0x06,
+0x09,
+0x2a,
+0x86,
+0x48,
+0x86,
+0xf7,
+0x0d,
+0x01,
+0x09,
+0x01,
+0x16,
+0x0f,
+0x62,
+0x65,
+0x6e,
+0x68,
+0x40,
+0x64,
+0x65,
+0x62,
+0x69,
+0x61,
+0x6e,
+0x2e,
+0x6f,
+0x72,
+0x67,
+0x30,
+0x1e,
+0x17,
+0x0d,
+0x31,
+0x38,
+0x30,
+0x34,
+0x31,
+0x33,
+0x31,
+0x38,
+0x32,
+0x31,
+0x34,
+0x36,
+0x5a,
+0x17,
+0x0d,
+0x31,
+0x38,
+0x30,
+0x35,
+0x31,
+0x33,
+0x31,
+0x38,
+0x32,
+0x31,
+0x34,
+0x36,
+0x5a,
+0x30,
+0x6b,
+0x31,
+0x0b,
+0x30,
+0x09,
+0x06,
+0x03,
+0x55,
+0x04,
+0x06,
+0x13,
+0x02,
+0x47,
+0x42,
+0x31,
+0x13,
+0x30,
+0x11,
+0x06,
+0x03,
+0x55,
+0x04,
+0x07,
+0x0c,
+0x0a,
+0x4d,
+0x61,
+0x6e,
+0x63,
+0x68,
+0x65,
+0x73,
+0x74,
+0x65,
+0x72,
+0x31,
+0x0f,
+0x30,
+0x0d,
+0x06,
+0x03,
+0x55,
+0x04,
+0x0a,
+0x0c,
+0x06,
+0x44,
+0x65,
+0x62,
+0x69,
+0x61,
+0x6e,
+0x31,
+0x16,
+0x30,
+0x14,
+0x06,
+0x03,
+0x55,
+0x04,
+0x03,
+0x0c,
+0x0d,
+0x42,
+0x65,
+0x6e,
+0x20,
+0x48,
+0x75,
+0x74,
+0x63,
+0x68,
+0x69,
+0x6e,
+0x67,
+0x73,
+0x31,
+0x1e,
+0x30,
+0x1c,
+0x06,
+0x09,
+0x2a,
+0x86,
+0x48,
+0x86,
+0xf7,
+0x0d,
+0x01,
+0x09,
+0x01,
+0x16,
+0x0f,
+0x62,
+0x65,
+0x6e,
+0x68,
+0x40,
+0x64,
+0x65,
+0x62,
+0x69,
+0x61,
+0x6e,
+0x2e,
+0x6f,
+0x72,
+0x67,
+0x30,
+0x82,
+0x01,
+0x22,
+0x30,
+0x0d,
+0x06,
+0x09,
+0x2a,
+0x86,
+0x48,
+0x86,
+0xf7,
+0x0d,
+0x01,
+0x01,
+0x01,
+0x05,
+0x00,
+0x03,
+0x82,
+0x01,
+0x0f,
+0x00,
+0x30,
+0x82,
+0x01,
+0x0a,
+0x02,
+0x82,
+0x01,
+0x01,
+0x00,
+0x9d,
+0xe1,
+0x77,
+0xa0,
+0x24,
+0xa0,
+0xd5,
+0x79,
+0x65,
+0x3a,
+0x07,
+0x90,
+0xc9,
+0xf6,
+0xa5,
+0xa6,
+0x1f,
+0x84,
+0x1c,
+0x23,
+0x07,
+0x4b,
+0x4f,
+0xa5,
+0x03,
+0xc6,
+0x0f,
+0xf7,
+0x54,
+0xd5,
+0x8b,
+0x7e,
+0x79,
+0x81,
+0x00,
+0xd2,
+0xe9,
+0x3d,
+0xf4,
+0x97,
+0xfe,
+0x84,
+0xcd,
+0x55,
+0xbd,
+0xc9,
+0x8f,
+0x21,
+0x57,
+0x88,
+0x06,
+0x39,
+0x90,
+0x66,
+0x41,
+0x26,
+0x79,
+0x2c,
+0xca,
+0x3f,
+0x95,
+0x87,
+0x01,
+0x11,
+0x2f,
+0x2f,
+0xb0,
+0xe1,
+0x0b,
+0x43,
+0xfc,
+0x5f,
+0x2f,
+0x4f,
+0x67,
+0x04,
+0xdb,
+0x4d,
+0xb7,
+0x72,
+0x4d,
+0xd1,
+0xc5,
+0x76,
+0x73,
+0x4d,
+0x91,
+0x69,
+0xb0,
+0x71,
+0x17,
+0x36,
+0xea,
+0xab,
+0x0a,
+0x3a,
+0xcd,
+0x95,
+0x9b,
+0x76,
+0x1b,
+0x8e,
+0x21,
+0x17,
+0x8f,
+0xc5,
+0x02,
+0xbf,
+0x24,
+0xc7,
+0xc0,
+0x40,
+0xb1,
+0x3b,
+0xc4,
+0x80,
+0x7c,
+0x71,
+0xa5,
+0x51,
+0xdc,
+0xf7,
+0x3a,
+0x58,
+0x7f,
+0xb1,
+0x07,
+0x81,
+0x8a,
+0x10,
+0xd1,
+0xf6,
+0x93,
+0x17,
+0x71,
+0xe0,
+0xfa,
+0x51,
+0x79,
+0x15,
+0xd4,
+0xd7,
+0x8f,
+0xad,
+0xbd,
+0x6f,
+0x38,
+0xe1,
+0x26,
+0x7d,
+0xbc,
+0xf0,
+0x3e,
+0x80,
+0x89,
+0xb4,
+0xec,
+0x8e,
+0x69,
+0x90,
+0xdb,
+0x97,
+0x8a,
+0xf0,
+0x23,
+0x23,
+0x83,
+0x82,
+0x3b,
+0x6a,
+0xb1,
+0xac,
+0xeb,
+0xe7,
+0x99,
+0x74,
+0x2a,
+0x35,
+0x8e,
+0xa9,
+0x64,
+0xfd,
+0x46,
+0x9e,
+0xe8,
+0xe5,
+0x48,
+0x61,
+0x31,
+0x6e,
+0xe6,
+0xfc,
+0x19,
+0x18,
+0x54,
+0xc3,
+0x1b,
+0x4f,
+0xd6,
+0x00,
+0x44,
+0x87,
+0x1c,
+0x37,
+0x45,
+0xea,
+0xf5,
+0xc9,
+0xcb,
+0x0f,
+0x0c,
+0x55,
+0xec,
+0xcf,
+0x6a,
+0xc2,
+0x45,
+0x26,
+0x23,
+0xa2,
+0x31,
+0x52,
+0x4d,
+0xee,
+0x21,
+0x7d,
+0xfd,
+0x58,
+0x72,
+0xc2,
+0x28,
+0xc5,
+0x8e,
+0xa9,
+0xd0,
+0xee,
+0x01,
+0x77,
+0x08,
+0xa5,
+0xf0,
+0x22,
+0x2b,
+0x47,
+0x79,
+0x2b,
+0xcf,
+0x9a,
+0x46,
+0xb5,
+0x8f,
+0xfd,
+0x64,
+0xa2,
+0xb5,
+0xed,
+0x02,
+0x03,
+0x01,
+0x00,
+0x01,
+0xa3,
+0x53,
+0x30,
+0x51,
+0x30,
+0x1d,
+0x06,
+0x03,
+0x55,
+0x1d,
+0x0e,
+0x04,
+0x16,
+0x04,
+0x14,
+0xd3,
+0xfb,
+0x2d,
+0xdb,
+0xf0,
+0x8e,
+0xfa,
+0x67,
+0x6d,
+0x2f,
+0x21,
+0x99,
+0x4c,
+0xeb,
+0x41,
+0x59,
+0xe6,
+0x9d,
+0xd8,
+0xd5,
+0x30,
+0x1f,
+0x06,
+0x03,
+0x55,
+0x1d,
+0x23,
+0x04,
+0x18,
+0x30,
+0x16,
+0x80,
+0x14,
+0xd3,
+0xfb,
+0x2d,
+0xdb,
+0xf0,
+0x8e,
+0xfa,
+0x67,
+0x6d,
+0x2f,
+0x21,
+0x99,
+0x4c,
+0xeb,
+0x41,
+0x59,
+0xe6,
+0x9d,
+0xd8,
+0xd5,
+0x30,
+0x0f,
+0x06,
+0x03,
+0x55,
+0x1d,
+0x13,
+0x01,
+0x01,
+0xff,
+0x04,
+0x05,
+0x30,
+0x03,
+0x01,
+0x01,
+0xff,
+0x30,
+0x0d,
+0x06,
+0x09,
+0x2a,
+0x86,
+0x48,
+0x86,
+0xf7,
+0x0d,
+0x01,
+0x01,
+0x0b,
+0x05,
+0x00,
+0x03,
+0x82,
+0x01,
+0x01,
+0x00,
+0x71,
+0x71,
+0xe4,
+0xcd,
+0x40,
+0x84,
+0xf3,
+0x60,
+0x34,
+0x6a,
+0x78,
+0xdc,
+0xe7,
+0x81,
+0x20,
+0x8f,
+0x1e,
+0x7f,
+0x54,
+0x31,
+0x8e,
+0x4a,
+0xe5,
+0x69,
+0x91,
+0x1c,
+0x4c,
+0x77,
+0x00,
+0x47,
+0x09,
+0x98,
+0x91,
+0xf4,
+0x27,
+0x52,
+0x9a,
+0x2d,
+0x30,
+0x87,
+0x4c,
+0x51,
+0x51,
+0x1f,
+0xae,
+0xd8,
+0x8c,
+0xdb,
+0x3c,
+0x73,
+0x06,
+0x43,
+0xc6,
+0x1f,
+0x3a,
+0xba,
+0xaf,
+0x9e,
+0xd2,
+0xeb,
+0x3e,
+0x9f,
+0xd1,
+0xb2,
+0x8a,
+0xa2,
+0x16,
+0x08,
+0x26,
+0xa1,
+0x39,
+0x0c,
+0x63,
+0xe7,
+0x2d,
+0x78,
+0x9c,
+0xac,
+0x2c,
+0x4e,
+0x69,
+0xd5,
+0xa0,
+0xfd,
+0xbd,
+0xea,
+0x95,
+0xfe,
+0xe2,
+0x69,
+0x06,
+0xe1,
+0xb2,
+0x27,
+0x90,
+0x68,
+0xd8,
+0x3a,
+0xac,
+0xd6,
+0xa0,
+0x4e,
+0xe2,
+0x8a,
+0xfa,
+0xff,
+0x9c,
+0x98,
+0x6b,
+0x60,
+0x83,
+0xea,
+0xe7,
+0x46,
+0x24,
+0x09,
+0x21,
+0x12,
+0xa8,
+0xfe,
+0xaa,
+0x46,
+0x0d,
+0x24,
+0xa5,
+0xcf,
+0x52,
+0x37,
+0x48,
+0xb9,
+0xe2,
+0xb1,
+0xd6,
+0xb0,
+0xe4,
+0xbf,
+0x6f,
+0x59,
+0x7b,
+0x17,
+0x9e,
+0xdd,
+0x24,
+0x00,
+0xee,
+0xc6,
+0x89,
+0x76,
+0xe9,
+0x35,
+0x40,
+0x1f,
+0xeb,
+0x7d,
+0x23,
+0x8c,
+0xed,
+0x13,
+0x6c,
+0x66,
+0xbc,
+0xc4,
+0x88,
+0xb5,
+0x55,
+0x9a,
+0xec,
+0xbd,
+0x52,
+0x0d,
+0x0b,
+0xc5,
+0x40,
+0xe1,
+0xfe,
+0xb7,
+0x20,
+0xb1,
+0x22,
+0x16,
+0x32,
+0xed,
+0x28,
+0x3e,
+0x1f,
+0xbf,
+0xf2,
+0x00,
+0x12,
+0x75,
+0x92,
+0xd7,
+0x99,
+0x2e,
+0x25,
+0xfb,
+0xf2,
+0xe6,
+0xfd,
+0x2a,
+0x10,
+0xd1,
+0x75,
+0x89,
+0x31,
+0x52,
+0xe4,
+0xb2,
+0xc2,
+0xee,
+0xcd,
+0x41,
+0xa3,
+0x08,
+0x48,
+0x18,
+0x5e,
+0x66,
+0x42,
+0x67,
+0xcf,
+0x0b,
+0x2a,
+0x26,
+0x6b,
+0x65,
+0x87,
+0x4d,
+0xfa,
+0x04,
+0x51,
+0xec,
+0xed,
+0x03,
+0x8b,
+0x38,
+0x52,
+0x93,
+0x6d,
+0xc4,
+0x30,
+0x41,
+0x9a,
+0x6c,
+0x28,
+0x35,
+0xf0,
+0x87,
+0x15,
+0xce,
+0x78,
+0x4f,
+0x32,
+0xca,
+0x52,
+0xed,
